sonar-csharp-plugin/src/main/java/org/sonar/plugins/csharp/CSharpMsBuildIntegrationProvider.java

```diff
@@ -37,7 +37,7 @@ public static List extensions() {
     return Collections.singletonList(
       PropertyDefinition.builder("sonar.cs.msbuild.testProjectPattern")
         .name("Test project pattern")
-        .description("Regular expression matched by test project files path (.NET syntax)")
+        .description("DEPRECATED - Use 'sonar.msbuild.testProjectPattern' key in the Scanner for MSBuild (client side) instead.\nRegular expression matched by test project files path (.NET syntax)")
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Split this 198 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00103)